src/lib/strings/handles.ts

```diff
@@ -3,7 +3,7 @@ export function makeValidHandle(str: string): string {
     str = str.slice(0, 20)
   }
   str = str.toLowerCase()
-  return str.replace(/^[^a-z]+/g, '').replace(/[^a-z0-9-]/g, '')
+  return str.replace(/[^a-z0-9-]/g, '')
```

Comment:
One tiny thing that might slip through here is handles beginning with a hyphen.

Reply:
Ah geez good call